src/middlewares/parse-query.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function parseQueryParams(request, response, next) {
+  if (!request.query.limit) request.query.limit = '50';
+  request.query.limit = parseInt(request.query.limit, 10);
+

```

Comment:
you can add extra validation for NaN result like 
`request.query.limit = parseInt(request.query.limit) | 50;`